KILTprotocol/demo-client
# feat: docker-compose file for easy setup

## relates to KILTProtocol/ticket#415

run `docker-compose up` to spin up a complete, containered development setup.
Changes in `/src` will be applied immediately, no need for a restart. Add with the `-d` flag to run in detached mode (background process, no log messages to console).
Stop and purge with `docker-compose down -v`.
A rebuild is necessary only if files outside of `/src` change (including dependencies). Trigger a rebuild with `docker-compose up --build`.

## How to test:
clone the repo in a fresh directory, start docker and run
`docker-compose up`
You should then be able to access the demo-client on `localhost:3001`

## Checklist:
